src/main/java/com/example/msgadminapi/service/UserService.java

```diff
@@ -66,4 +72,23 @@ private List<UserResponseDto> getUserResponseDtoList(List<User> all) {
                 .build()));
         return searchList;
     }
+
+    public JwtResponseDto login(String email) {
+        User user = userRepository.findByEmail(email)
+                .orElseThrow(() -> new RuntimeException());
+        if(user==null) {
+            throw new UserNotFindException("User can't find", ErrorCode.USER_NOT_FIND);
+        }
```

Comment:
ofElseThrow로 null을 검사 했는데 다시 user를 검증할 필요가 없어보여요
```suggestion
                .orElseThrow(() -> new RuntimeException());
```

Reply:
해결했습니다.

Reply:
[링크](https://github.com/GSM-MSG/GCMS-ADMIN-Backend/pull/26/commits/da2bfe4bc3da948e7cb99e0ed0f00a312088c33f)